docs/en/rules/Azure.ACR.Firewall.md

```diff
@@ -0,0 +1,109 @@
+---
+severity: Important
+pillar: Security
+category: Application endpoints
+resource: Container Registry
+online version: https://azure.github.io/PSRule.Rules.Azure/en/rules/Azure.ACR.Firewall/
+---
+
+# Restrict network access
+
+## SYNOPSIS
+
+Restrict network access.
+
+## DESCRIPTION
+
+Azure Container Registry (ACR) allows you to restrict network access.
+Network restriction can be implemented by firewall rules or private endpoints.
+
+A container registry should accept explicitly allowed traffic only.
+
+However, no network access restriction can be used in scenarios that do not require network restriction such as distributing public container images.
```

Comment:
```suggestion
Azure Container Registry (ACR) allows you to restrict network access to trusted clients and networks instead of any client.

Container registries using the Premium SKU can limit network accesss by setting firewall rules or using private endpoints.
Firewall and private endpoints are not supported when using the Basic or Standard SKU.

In general, network access should be restricted to harden against unauthorized access or exfiltration attempts.
However may not be required when publishing and distributing public container images to external parties.
```